NightscoutUploadKit/NightscoutUploader.swift

```diff
@@ -122,20 +122,6 @@ public class NightscoutUploader {
         uploadToNS(treatments, endpoint: defaultNightscoutTreatmentPath, completion: completionHandler)
     }
 
-    public func getPumpStatusFromMySentryPumpStatus(status: MySentryPumpStatusMessageBody) throws -> PumpStatus {
```

Comment:
Why did you remove this, exactly? Is there an alternative method for uploading these values from MySentry records, or were you thinking you wanted all pumps to wake+poll (which I, personally, would not want to do)